src/homework1/exam15.py

```diff
@@ -0,0 +1,19 @@
+# 15 lines: itertools
+
+
+from itertools import groupby
+
```

Comment:
here you have 1 line between import and code (previously 2 lines in other tasks). It should be consistent in you code